econml/_ortho_learner.py

```diff
@@ -899,7 +899,7 @@ def score(self, Y, T, X=None, W=None, Z=None, sample_weight=None, groups=None):
                     nuisances = [np.zeros((n_iters * n_splits,) + nuis.shape) for nuis in nuisance_temp]
 
                 for it, nuis in enumerate(nuisance_temp):
-                    nuisances[it][i * n_iters + j] = nuis
+                    nuisances[it][j * n_iters + i] = nuis
```

Comment:
Should we link this to the inciting issue.

Reply:
The name of that commit mentions it, and I'll rebase rather than squash when merging so that it will be maintained.